addOns/commonlib/src/test/java/org/zaproxy/addon/commonlib/PolicyTagsUnitTest.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Zed Attack Proxy (ZAP) and its related class files.
+ *
+ * ZAP is an HTTP/HTTPS proxy for assessing web application security.
+ *
+ * Copyright 2024 The ZAP Development Team
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.zaproxy.addon.commonlib;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.is;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.junit.jupiter.api.Test;
+import org.zaproxy.addon.commonlib.tags.PolicyTags;
+
+class PolicyTagsUnitTest {
+
+    @Test
+    void allPolicyTagsShouldStartWithPolicyUnderscoreInCaps() {
```

Comment:
`shouldHaveAllNamesStartingWithPolicy…` though this does not need to be in the name given that the enum name already says that, it can concatenate that in the getTag (or even add at build time through the constructor).

Reply:
That's what was agreed in slack

Reply:
Why do further processing after the fact? That seems inefficient.

Reply:
The tags not the enum names?

Reply:
We def want them to start qith `POLICY_` on https://www.zaproxy.org/alerttags/ - is thats the same thing?